cinema/serializers.py

```diff
@@ -1 +1,120 @@
 # write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name",)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField(
+        source="full_name",
+        read_only=True
+    )
+
+    class Meta:
+        model = Actor
+        fields = (
+            "id",
+            "first_name",
+            "last_name",
+            "full_name",
+        )
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
```

Comment:
You can use str(obj) instead of f"{obj.first_name} {obj.last_name}" since str method redefined in Actor model

Reply:
Good mark